R/AttributeCharts.R

```diff
@@ -120,6 +120,16 @@ if(options$Attributes == "Defectives"){
   if (!ready)
     return()
 
+  .hasErrors(
```

Comment:
is it necessary to make 2 identical calls to the function `.hasErrors`? Why no create a function, say `.attributeChartsErrorCheck`, and then call that function once before you call these two functions?

Reply:
I can make a function and then call it twice, but this check is only necessary for these two specific charts, not to the entire analysis.